content/articles/transferring-domain.markdown

```diff
@@ -60,7 +61,7 @@ Your domain transfer request will now be issued if there are no missing fields.
 
 Follow the instructions in the email. We cannot issue the transfer request to the registry without approval of this email.
 
-Once you've authorized the transfer, you may have to **wait up to 7 days your domain transfer to complete.**
+Once you've authorized the transfer, you may have to **wait up to 5 days your domain transfer to complete.**
```

Comment:
@OleMchls @kayla-broussard FYI there is a reason why we write 7 days. While ICANN says 5 days, systems don't work with second-precision.

Sometimes it's 5 days an 1 hour, sometimes 5 days and many hours (closer to 6). If we factor in time-zones (someone starting at the end of their day, then checking at the beginning of the 5th day), along with delays, the perception is sometimes effectively 6 to 7 days.

We explicitly changed from 5 to 7 in the past due to the number of support requests. We should not revert back.

Reply:
@weppos I've adjusted this in 31df3de 👍